devices.js

```diff
@@ -8511,6 +8511,39 @@ const devices = [
             }
         },
     },
+    {
+        zigbeeModel: ['URC4450BC0-X-R'],
+        model: 'URC4450BC0-X-R',
+        vendor: 'Xfinity',
+        description: 'Alarm Keypad',
+        supports: 'action, arm',
+        meta: {configureKey: 1, commandArmIncludeTransaction: true},
+        fromZigbee: [fz.command_arm, fz.temperature, fz.battery, fz.ias_occupancy_alarm_1, fz.identify, fz.ias_contact_alarm_1],
+        exposes: [e.battery(), e.voltage(), e.occupancy(), e.battery_low(), e.tamper(), e.presence(), exposes.numeric('action_code', 'r'),
+            exposes.numeric('action_zone', 'r'), e.temperature(), e.action([
+                'disarm', 'arm_day_zones', 'identify', 'arm_night_zones', 'arm_all_zones', 'invalid_code', 'emergency',
```

Comment:
Please add `set_status` here, should be something like:


```js
new Composite('set_status', 'set_status')
  .withFeature(new exposes.Numeric('seconds_remaining', access.SET))
  .withFeature(new exposes.Numeric('alarm_status', access.SET))
  .withFeature(new exposes.Enum('panel_status', access.SET, ['arm', '...... more'])),
  . withDescription('Set status of the panel');
```

Reply:
I've tried to add but get the following error

![image](https://user-images.githubusercontent.com/14173108/101420247-bbfa0100-392c-11eb-8882-10140480b096.png)

Reply:
Fixed in latest dev. (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

Reply:
I've switched to dev branch and it now starts, however no controls are presented in home assistant and in the zigbee2mqtt web interface it shows the following for exposes

![image](https://user-images.githubusercontent.com/14173108/101627721-bcdc8100-3a59-11eb-9bd8-94a5e28f44b6.png)

Reply:
@nurikk I think composite is not supported in the frontend yet (since we did not use it before). Can this be added?

Reply:
Just merged into dev branch, welcome to test

Reply:
It now comes up in the exposes page and i can set the status from that page, still nothing in the actual home assistant entities section though
![image](https://user-images.githubusercontent.com/14173108/101764612-34261980-3b1b-11eb-9d3e-3cedb6b7911c.png)

Reply:
Home Assistant does not supported such advanced things so it will only be available through the frontend.

Reply:
Ok will have to wait until it is.